library/Zend/Db/Select.php

```diff
@@ -1185,6 +1214,19 @@ protected function _renderFrom($sql)
 
         return $sql;
     }
+    
+    /**
+     * Render FORCE INDEX option
+     *
+     * @param $sql
+     * @see Zend_Db_Select::_getForceIndexForTable
+     * @return string
+     */
+    protected function _renderForceIndex($sql)
```

Comment:
Not sure why this is here, could you please remove that? I would like to merge this one, thanks.